app/controllers/favorites_controller.rb

```diff
@@ -8,4 +8,11 @@ def create
       format.json { render json: fp.save.to_json }
     end
   end
+
+  def destroy
+  	@favorite_project = FavoriteProject.where(project_id: params[:id],user_id: current_user.id).first
```

Comment:
Line is too long. [100/80]<br>Use 2 (not 1) spaces for indentation.<br>Space missing after comma.<br>Tab detected.